dcli/template/kotlin/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+echo "Compiling kotlin code"
+cd src/main/kotlin
+kotlinc -Xmaxerrs 1 app/*.kt
```

Comment:
Kotlin is not present within your `Dockerfile`, check the following example: https://github.com/deadlock-resources/challenge-examples/tree/master/example/code_kotlin

Reply:
Done